docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -104,21 +124,65 @@ However there are some limitations when using external packages:
 * Native node modules will not work as expected due to platform inconsistency. 
 ====
 
-`--auth`::
+`--auth <string>`::
 API key used for {kibana-ref}/api-keys.html[{kib} authentication].
 +
 If you are pushing to a <<private-locations,{private-location}>>, you must use an API key generated in 8.4 or higher.
 +
 To create an API key, you must be logged into {kib} as a user with the privileges described in
 <<synthetics-role-write>>.
 
-`--url`::
-The {kib} URL for the deployment to which you want to upload the monitors.
-
-`--project`::
+`--project <string>`::
 A unique id associated with your project.
 It will be used for logically grouping monitors.
++
 If you used <<elastic-synthetics-init-command, `init` to create a project>>, this is the `<name-of-project>` you specified.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-project,`project.id`>>.
+The value defined via the CLI will take precedence.
+
+`--url <string>`::
+The {kib} URL for the deployment to which you want to upload the monitors.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-project,`project.url`>>.
+The value defined via the CLI will take precedence.
+
+`--space <string>`::
+The identifier of the target {kibana-ref}/xpack-spaces.html[{kib} space] for the pushed monitors.
+Spaces help you organize pushed monitors.
+Defaults to `default`.
```

Comment:
Maybe its just me, I find it better to say like `Pushes to "default" space if unspecified`